generators/gae/index.js

```diff
@@ -141,8 +142,93 @@ module.exports = class extends BaseGenerator {
         return [_.kebabCase(this.baseName)];
     }
 
+    getMicroserviceFolders(input) {
+        if (!input) return null;
+        const destinationPath = this.destinationPath(input);
+        const files = shelljs.ls('-l', destinationPath);
+        const appsFolders = [];
+
+        files.forEach(file => {
+            if (file.isDirectory()) {
+                if (shelljs.test('-f', `${destinationPath}/${file.name}/.yo-rc.json`)) {
```

Comment:
I prefer ```fs.existsSync(`${destinationPath}/${file.name}/.yo-rc.json`)``` here.

Using shelljs should be avoided.

Reply:
Thanks. Done. :+1: